CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * [Fix] Display a message instead of an error when `toml` isn't installed and `pyproject.toml` is found (#825)
 * [Fix] Fix argument parsing error on Windows when it contains quotations (#425)
 * [Fix] Fix error when a linebreak is included during nonidentifier validation process
+* [Fix] Fix `isolate_tests` function to ensure stored snippets are cleared (#820)
```

Comment:
since this is an internal change, we don't put it in the changelog. we only put things that affect end users, I added the "no changelog" label to this PR so the CI passes even if the changelog has no changes

Reply:
fixed